code401challenges/src/main/java/sortingAlgorithms/MergeSort.java

```diff
@@ -0,0 +1,50 @@
+package sortingAlgorithms;
+
+public class MergeSort {
+    public int[] mergeSort(int[] arr) {
+        int n = arr.length;
+        int mid;
+        int[] left;
+        int[] right;
+
+        if (n < 1) {
```

Comment:
You could put this before most of the variable declarations. Why create all those variable if you're just going to return the array?